test/lru_cache_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+class TestLRUCache < Minitest::Test
+  def setup
+    @lru_cache = LRUCache.new(5)
+  end
```

Comment:
You need a test with a bulkhead too, to ensure those don't get deleted.

Reply:
Done!